File: haskell/src/CLI/Function/Random.hs
module CLI.Function.Random (printRandomNumbers) where

import Numbers.Utils.Operations.FactorialUtils (factorialRandom)
import Numbers.Utils.Operations.FibonacciUtils (fibonacciRandom)

-- |
-- = @printRandomNumbers@
--
-- The function @printRandomNumbers@ is used to print the random fibonacci and
-- factorial numbers by calling the @fibonacciRandom@ and @factorialRandom@.
printRandomNumbers :: IO ()
printRandomNumbers = do
    putStrLn "Some random numbers : "
    putStrLn =<< fibonacciRandom
    putStrLn =<< factorialRandom

File: kotlin/packages/cli-function/bin/main/com/prog/cli/function/Random.kt
package com.prog.cli.function

import com.prog.numbers.utils.operations.FactorialUtils.factorialRandom
import com.prog.numbers.utils.operations.FibonacciUtils.fibonacciRandom

object Random {
    /**
     * # `main`
     *
     * This function is the entry point of the application.
     *
     * __Arguments:__
     */
    @JvmStatic
    fun main(args: Array<String>) {
        println("===================================================")
        printRandomNumbers()
        println("===================================================")
    }

    /**
     * # `printRandomNumbers`
     *
     * This function prints random fibonacci and factorial by calling the
     * `fibonacciRandom` and `factorialRandom` functions respectively.
     */
    fun printRandomNumbers() {
        println(fibonacciRandom())
        println(factorialRandom())
    }
}
